src/main/java/Duke.java

```diff
@@ -1,10 +1,85 @@
+        System.out.println("Hello! I'm Duke \nWhat can I do for you?");
+        Scanner in = new Scanner(System.in);
+        ArrayList<Task> taskList = new ArrayList<Task>();
+
+        String userInput;
+        while(in.hasNext()){
+
+            userInput = in.nextLine();
+            if(userInput.equals("bye")){
+                break;
+            }
+            if(userInput.equals("list")) {
+                for (int i = 0; i < taskList.size(); i++) {
+                    System.out.print(i + 1 + ".");
+                    System.out.println(taskList.get(i).toString());
+                }
+            } else if (userInput.contains("todo")) {
+                String info = userInput.substring(5).trim();
+                taskList.add(new Todo(info));
+                int finalTask = taskList.size() - 1;
+                System.out.println("Got it. I've added this task:");
+                System.out.println(taskList.get(finalTask).toString());
+                if (taskList.size() == 1){
+                    System.out.println("Now you have " +taskList.size()+" task in the list.");
+                } else {
+                    System.out.println("Now you have " + taskList.size() + " tasks in the list.");
+                }
+            } else if (userInput.contains("deadline")) {
+                String [] listArray = userInput.split("/",2);
+                String description = listArray[0];
+                String dueDate = listArray[1];
+                String info = description.substring(8).trim();
+                String due = dueDate.substring(3).trim();
+                taskList.add(new Deadline(info, due));
+                int finalTask = taskList.size() - 1;
+                System.out.println("Got it. I've added this task:");
+                System.out.println(taskList.get(finalTask).toString());
+                if (taskList.size() == 1){
+                    System.out.println("Now you have " +taskList.size()+" task in the list.");
+                } else {
+                    System.out.println("Now you have " + taskList.size() + " tasks in the list.");
+                }
+            } else if (userInput.contains("event")) {
+                String [] listArray = userInput.split("/",3);
+                String description = listArray[0];
+                String startTime = listArray[1];
+                String endTime = listArray[2];
+                String info = description.substring(6).trim();
+                String start = startTime.substring(5).trim();
+                String end = endTime.substring(3).trim();
+                taskList.add(new Event(info, start, end));
+                int finalTask = taskList.size() - 1;
+                System.out.println("Got it. I've added this task:");
+                System.out.println(taskList.get(finalTask).toString());
+                if (taskList.size() == 1){
+                    System.out.println("Now you have " +taskList.size()+" task in the list.");
```

Comment:
Perhaps you can include spaces like this to be more consistent in your formatting
```suggestion
                    System.out.println("Now you have " + taskList.size() + " task in the list.");
```